TOPdeskPS/functions/Change Management/Get-TdChangeCalendar.ps1

```diff
@@ -0,0 +1,121 @@
+﻿function Get-TdChangeCalendar {
+    <#
+.SYNOPSIS
+    returns list of changes and activities in specified date range
+.DESCRIPTION
+    returns list of changes and activities in specified date range. For activities to appear in this endpoint the option “Display in Calendar” under “Planning” on the activity card has to be set. Start and end date are mandatory.
+.PARAMETER Start
+    start of range to return changes. Default value = 90 days
+.PARAMETER End
+    end of range to return changes. Default value = 90 days
+.PARAMETER CardType
+    Type of card to filter results by
+.PARAMETER Branch
+    Branch to filter results by
+.PARAMETER Category
+    Category of cards to obtain
+.PARAMETER Status
+    Status of cards to obtain
+.PARAMETER Type
+    Type of cards to obtain
+.PARAMETER CurrentState
+    CurrentState that cards to be obtained are in
+.EXAMPLE
+    PS C:\> Get-TdChangeCalendar
+    Returns results with default date values (90 days back and 90 days forward)
+#>
```

Comment:
add another example or 2 showing off a couple of parameters

Reply:
(y)